### DISCUSSION TITLE
Can PAC tool export any Custom Code “widgets” from a Model Driven app or anything we tag specifically [eg, .js] ?

### DISCUSSION THREAD

**Discussion by gituseranh** [microsoft/powerplatform-build-tools#946] [Q&A]
Can the PAC tool be used to scan solutions for web resources of specific file types (eg, .js) and plug-in assemblies ( .NET) in order to make copies of those files to a folder?  What would be the command syntax for that in pac?

**Comment by devkeydet** [comment]
`pac solution unpack`, `pac solution clone`, `pac solution sync` will all unpack a solution.  In the case of web resources, it will unpack the js file.  In the case of plugin assemblies, you just get the dll.  You will want to look at cdsproj and its support for building code first assets and then building them into a solution zip as part of the build process.  Here are some resources to get you started understanding the coordination between low code and code first assets through cdsproj.
https://github.com/devkeydet/HelloFusionDev 
https://youtu.be/wzHLWNWcY2Q

**Reply by gituseranh** [reply]
Thanks very much for the response ... will explore the links provided. Really appreciate it.
